### ISSUE TITLE
hidpi buttons

### ISSUE THREAD

**Issue by myf** [aidanholm/luakit#410]
so the way i am dealing with hidpi screen is to set zoom_level=2.0 in globals.lua, and view.zoom_level = zoom_level or 2.0 in webview.lua. 
It has been working well for me, only one slight annoyance: buttons. buttons seems to be zoomed properly even without the config zoom_level set up (perhaps it read the dpi from the system?) and when I zoom everything 2x buttons would look like 4x. here are two comparison screenshots, one taken from zoom_level = 1.0 and another from zoom_level = 2.0.
![2017-08-09-152324_2560x1700_scrot](https://user-images.githubusercontent.com/694376/29140421-5ff76b78-7d18-11e7-9413-4152ee920220.png)
vs.
![2017-08-09-152358_2560x1700_scrot](https://user-images.githubusercontent.com/694376/29140424-6597224e-7d18-11e7-813a-6ef40b60c869.png)

i wonder what the issue would be. could it be my own config or something has to do with how webkit treats buttons?

**Comment by aidanholm** [comment]
Right, that's a bit annoying. Almost certainly a WebKitGTK issue. I'm guessing it's the same with other sites, not just reddit?

BTW I'm working on the main luakit/luakit repository now, and I'll eventually stop pushing to this one; it's behind already, I think.

**Comment by myf** [comment]
ah good to know! I'll migrate this over
